src/ILCompiler.ReadyToRun/src/Compiler/DependencyAnalysis/ReadyToRun/ImportThunk.cs

```diff
@@ -30,13 +30,13 @@ enum Kind
 
         private readonly Kind _thunkKind;
 
-        public ImportThunk(ReadyToRunHelper helperId, ReadyToRunCodegenNodeFactory factory, Import instanceCell)
+        public ImportThunk(ReadyToRunHelper helperId, ReadyToRunCodegenNodeFactory factory, Import instanceCell, bool useVSD)
```

Comment:
`VSD` is a cryptic TLA. Would it be better to call this `virtualCall`?

Reply:
Makes sense, renamed in 2nd commit.